src/tests/build-test.js

```diff
@@ -60,5 +60,5 @@ function simpleTest(url) {
     })
 }
 
-simpleTest('/fishes.jpg')
+//simpleTest('/fishes.jpg')
 saveInc('/fishes.jpg', glitchParams)
```

Comment:
This function is partially succesful https://github.com/kalwalt/visualARPoetry-backend/runs/2333124783?check_suite_focus=true 
the **saveInc()**  pass but with this message:

```
Run npm run build-test --if-present
  npm run build-test --if-present
  shell: /usr/bin/bash -e {0}

> visual-ar-poetry-backend@0.1.0 build-test /home/runner/work/visualARPoetry-backend/visualARPoetry-backend
> node src/tests/build-test.js

[Arguments] {
  '0':
   Error: Stream yields empty buffer
       at Socket.<anonymous> (/home/runner/work/visualARPoetry-backend/visualARPoetry-backend/node_modules/gm/lib/command.js:57:17)
       at Socket.emit (events.js:203:15)
       at endReadableNT (_stream_readable.js:1145:12)
       at process._tickCallback (internal/process/next_tick.js:63:19),
  '1': null }
```
Instead all jest tests are completely succesfull.